man/man7/vdevprops.7

```diff
@@ -156,6 +156,8 @@ If this device should perform new allocations, used to disable a device
 when it is scheduled for later removal.
 See
 .Xr zpool-remove 8 .
+.It Sy queue_io
+Add io to the vdev queue when reading or writing to this vdev.
```

Comment:
Might want to give a hint to the user about when they would want to change this prop:
```
Add IO to the vdev queue when reading or writing to this vdev.
Disabling this property can sometimes improve performance for direct IOs.
```
Feel free to re-word this :arrow_up:

Reply:
Added.